package.json

```diff
@@ -39,7 +39,7 @@
     "mimelib": "0.2.14",
     "mocha": "1.17.1",
     "mongojs": "2.4.0",
-    "mongoose": "4.1.0",
+    "mongoose": "4.9.8",
```

Comment:
Is it worth making the cool down and mongoose upgrade separate deploys so if this PR fixes the issue we know if it was mongoose or the cool down code?

We will need to give the site a really good test with the new mongoose.